sycl/test/abi/pi_level_zero_symbol_check.dump

```diff
@@ -70,6 +70,7 @@ piProgramRelease
 piProgramRetain
 piQueueCreate
 piQueueFinish
+piQueueFlush
```

Comment:
Is there anything like this for OpenCL plugin?

Reply:
There is: https://github.com/intel/llvm/blob/sycl/sycl/test/abi/pi_opencl_symbol_check.dump. However, this patch does not introduce any new symbols to the OpenCL plugin library, so the check passes as-is.